File: Python/Math/Corner Digits Sum.py
//https://www.geeksforgeeks.org/problems/corner-digits1317/1
class Solution:
	def corner_digitSum(self, n):
		# Code here
		sum_ = n % 10
        count = 0
        while n > 0:
            if n == n % 10 and count > 0:
                sum_ += n % 10
            count += 1
            n = n // 10
        return sum_

File: Cpp/Math/Corner Digits Sum.cpp
//https://www.geeksforgeeks.org/problems/corner-digits1317/1
class Solution{
	public:
   	int corner_digitSum(int n){
   	    // Code here
   	    int sum = n%10, count = 0;
        while(n>0){
            if(n==n%10 && count>0){
                sum = sum+(n%10);
            }
            count++;
            n=n/10;
        }
        return sum;
   	}    
};
